data/alphaAma.com.md

```diff
@@ -0,0 +1,30 @@
+---
+title: AlphaAMA.com
+web: https://alphaama.com/
+git: https://github.com/eskema/alphaama
+instances:
+- name: AlphaAMA
+  url: https://alphaama.com
+  comment: Instance managed by AlphaAMA's author.
+tags:
+- client
+platforms:
+- web
+license: 
+progLang:
+- JS
+features:
+- kind1/post
+- reply
+- mentions
+- reactions
```

Comment:
These features are supported by most other social clients, too. Until we have the time to list all these details, we should not mark some as such. These are implied by "social".

Reply:
removing.